server/container_create_linux.go

```diff
@@ -969,6 +969,16 @@ func addOCIBindMounts(ctx context.Context, ctr ctrfactory.Container, mountLabel,
 		if m.HostPath == "/" && dest == "/" {
 			log.Warnf(ctx, "Configuration specifies mounting host root to the container root.  This is dangerous (especially with privileged containers) and should be avoided.")
 		}
+
+		hostPaths := []string{"/var", "/var/lib", "/var/lib/containers", "/lib", "/"}
+
+		for _, path := range hostPaths {
```

Comment:
another nit: can we name this variable something more informative? `pathsContainingStorageRoot` or something?